src/pages/about.js

```diff
@@ -121,10 +121,16 @@ const AboutPage = ({ data }) => {
 
           <p>Our team includes:</p>
 
-          {/* Show team members with photos in columns */}
+          {/*
+           * Show team members with photos in columns
+           *
+           * Accessibility note: All images are marked decorative since we list
+           * each team member's name right below, so saying "Photo of Name" is
+           * not very useful
+           */}
           <div className="team-photos mt-4 mb-3">
             <div className="text-center">
-              <img src={DerekPhoto} className="rounded img-fluid" />
+              <img src={DerekPhoto} alt="" className="rounded img-fluid" />
```

Comment:
why are we including empty `alt` tags?

Reply:
@derekeder - an empty alt tag is a marker that an image is decorative (basically intentionally not having an alt tag) and can be ignored. This is typically used for background images or visual breaks, but in this case since our names are right below each image, the image is acting in just a decorative sense.

I think technically you could make an alt tag for each image where you visually describe each of us, but I don't think that's really super valuable personally 🤷🏻‍♂️

Reply:
good to know. and the absence of an `alt` tag causes different behavior in screen readers?

Reply:
Yep! Without an alt tag a screen reader thinks it's a mistake and will read the file name or try to guess what the image contents are

Reply:
good to know!